stackgl_modules/package.json

```diff
@@ -13,21 +13,22 @@
     "box-intersect": "plotly/box-intersect#v1.1.0",
     "convex-hull": "^1.0.3",
     "delaunay-triangulate": "^1.1.6",
-    "gl-axes3d": "^1.7.0",
-    "gl-cone3d": "^1.6.0",
-    "gl-error3d": "^1.0.16",
+    "gl-axes3d": "github:hborchardt/gl-axes3d#fcc1c5c49db3de6445020a63a5d76403ec5a2b98",
```

Comment:
See https://github.com/gl-vis/gl-axes3d/pull/23

Reply:
Please install the patch.
```
npm i gl-axes3d@1.7.1
```